CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- [GH-37]: `-keyString` replaced by `-clientId` and `-clientSecret`
```

Comment:
I would make a more generic statement i.e. update list of arguments, because there were more arguments renamed than just `keyString`

Reply:
Ok

Reply:
see https://github.com/030/bcbsn/pull/39